### PR TITLE
Change `workload_id` type to `uint64`

### PR DESCRIPTION
**Pull request by panh99**
Changed data type and corresponding unit tests.

### CODE REVIEW

File: src/py/flwr/server/state/sqlite_state.py [hunk 1]
```diff
@@ -498,11 +498,10 @@ def get_nodes(self, workload_id: str) -> Set[int]:
         result: Set[int] = {row["node_id"] for row in rows}
         return result
 
-    def create_workload(self) -> str:
+    def create_workload(self) -> int:
         """Create one workload and store it in state."""
         # String representation of random integer from 0 to 9223372036854775807
```
Comment (charlesbvll) [inline src/py/flwr/server/state/sqlite_state.py:503]: This should be removed or modified

File: src/py/flwr/server/state/in_memory_state.py [hunk 2]
```diff
@@ -206,14 +206,13 @@ def get_nodes(self, workload_id: str) -> Set[int]:
             return set()
         return self.node_ids
 
-    def create_workload(self) -> str:
+    def create_workload(self) -> int:
         """Create one workload."""
         # String representation of random integer from 0 to 9223372036854775807
```
Comment (charlesbvll) [inline src/py/flwr/server/state/in_memory_state.py:211]: This should be removed or modified

File: src/py/flwr/server/state/sqlite_state.py (hunk 1 above)
Comment (panh99) [reply src/py/flwr/server/state/sqlite_state.py:503]: yes, thanks for pointing it out

**Review by charlesbvll — APPROVED** [review]

**Review by danieljanes — APPROVED** [review]
Lgtm! 🙌

**Pull request merged**